cmsplugin_filer_link2/admin.py

```diff
@@ -1,22 +1,59 @@
 # -*- coding: utf-8 -*-
+from django.conf.urls import url
 from django.contrib import admin
+from django.shortcuts import redirect
 from django.utils.safestring import mark_safe
 from django.utils.translation import activate
 from django.utils.translation import ugettext as _
 
+from cmsplugin_filer_link2.management.commands import check_links
 from .models import LinkHealthState
 
 
 class LinkStateAdmin(admin.ModelAdmin):
```

Comment:
The `link_name` in the list currently links to the link state of a link. There, the user could change the link or the status. I think both doesn't make really sense, maybe we should disable linking to the health status from within the list?

This should do the trick from within the `__init__` of the `LinkStateAdmin`:
`self.list_display_links = (None, )`